### ISSUE TITLE
Few type related questions

### ISSUE THREAD

**Issue by fluxxus** [dhg/Skeleton#230]
Is there any rationale behind type styles for headings? Is it based on some modular scale or just by what feels and looks right?

Also, what is the deal with negative letter spacing for h1-h6? :))

**Comment by stephnr** [comment]
I would imagine the letter spacing is for effect. Try toggling the spacing. It does have a strong affect on the letters anti-aliasing.

**Comment by fluxxus** [comment]
Yes, it does have an effect. But I think it is one of those things that shouldn't be there by default. There are many typefaces available, multiiple platforms and browsers, it may not create the same effect everywhere.

**Comment by stephnr** [comment]
I do not believe this project was initially designed to work with different kinds of typefaces. But, that does not mean we can consider the possibility! I say give it a shot
